src/components/atoms/dialog/DialogColorPicker.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { SketchPicker, ColorResult } from 'react-color'
+import {
+  DialogColorPickerPopover,
+  DialogColorPickerCover,
+  DialogColorPickerContainer,
+} from './styled'
+import { defaultTagColor, getColorBrightness } from '../../../lib/colors'
+
+type ColorPickerProps = {
+  color?: string
+  handleChangeComplete: (color: string) => void
+}
+
+type ColorPickerStates = {
+  color: string
+  colorBrightness: number
+}
+
+export default class DialogColorPicker extends React.Component<
```

Comment:
I don't see any benefits to make a class component. Please covert this into a function component!

Reply:
True, refactored